docs/cloud-messaging/receive.md

```diff
@@ -410,40 +411,39 @@ experimentalSetDeliveryMetricsExportedToBigQueryEnabled(messaging, true);
 
 Don't forget to run `yarn build` in order to export the new version of your service worker to the `web` folder.
 
+## Display images in notifications on iOS
 
-## (Advanced, Optional) Allowing Notification Images on iOS
-
-On Apple devices, in order for incoming FCM Notifications to display images from the FCM payload, you must add an additional notification service extension. This is not a required step.
+On Apple devices, in order for incoming FCM Notifications to display images from the FCM payload, you must add an additional notification service extension and configure your app to use it.
 
-If you are using FirebaseAuth phone authentication, you must add the Firebase Auth pod to your Podfile.
+If you are using Firebase phone authentication, you must add the Firebase Auth pod to your Podfile.
 
 ### Step 1 - Add a notification service extension
 
-- From Xcode top menu go to: File > New > Target...
-- A modal will present a list of possible targets, scroll down or use the filter to select "Notification Service Extension". Press Next.
-- Add a product name (use ImageNotification to follow along), set Language to Objective-C and click Finish.
+- From XCode top menu go to: **File > New > Target...**
+- A modal will present a list of possible targets, scroll down or use the filter to select "Notification Service Extension". Click Next.
```

Comment:
Hate to say it, but the bolding of UI elements applies even to "Next" and any other label on the screen.  

Point 8 in the contributing guidelines cites the style guide requirement, and the specific guideline here is https://developers.google.com/style/ui-elements#formatting.

That's for all UI elements in the PR, globally.